custom/live-streaming/components/LiveStreamingModal.js

```diff
@@ -116,15 +123,17 @@ export const LiveStreamingModal = () => {
           </Field>
         )}
 
-        {layout ===
-          LAYOUTS.findIndex((l) => l.value === 'single-participant') && (
+        {layoutType === 'single-participant' && (
           <Field label="Select participant">
             <SelectInput
-              onChange={(e) => setParticipant(e.target.value)}
-              value={participant}
+              onChange={(e) => setParticipantId(e.target.value)}
+              value={participantId}
             >
-              {allParticipants.map((p) => (
-                <option value={p.id} key={p.id}>
+              <option value={0} disabled>
+                Select
+              </option>
+              {participants.map((p) => (
+                <option value={p.sessionId} key={p.sessionId}>
                   {p.name}
                 </option>
               ))}
```

Comment:
Let's make a little UX improvement while we're here: Can you link to https://docs.daily.co/guides/paid-features/live-streaming-with-daily in the Enter RTMP endpoint label. This will help anyone using Daily live streaming for the first time :) 

Enter RTMP header ([Learn more](https://docs.daily.co/guides/paid-features/live-streaming-with-daily))